development/src/components/peerId_comp.js

```diff
@@ -0,0 +1,76 @@
+'use client'
+import React, { useState, useEffect } from 'react';
+import Image from 'next/image';
+import { generatePeerIdCharacter } from '../../src/composables/peerIdGenerator';
+
+import closeIconBlack from '../../public/assets/onboardingIcons/close_black.png';
+import closeIconWhite from '../../public/assets/onboardingIcons/close_light.png';
+
+function PeerId() {
+    const [isOpen, setIsOpen] = useState(true);
+    const [peerSessionId, setPeerSessionId] = useState('');
+
+    useEffect(() => {
+        setPeerSessionId(generatePeerIdCharacter()); 
+    }, []);
+
+    const handleClose = () => {
+        setIsOpen(false);
+    };
+
+    const handleCopyId = async () => {
+        try {
+            await navigator.clipboard.writeText(peerSessionId);
+            console.log('ID copied to clipboard:', peerSessionId);
+        } catch (error) {
+            console.error('Failed to copy ID to clipboard:', error);
+        }
+    };
+
+    if (!isOpen) {
+        return null;
+    }
+
+    return (
+        <div className="bg-white text-[#0E0C46] dark:bg-[#504F5F]  dark:text-white p-4 flex flex-col rounded-lg">
+            <div className="flex w-full justify-between">
+                <div className="font-nohemi font-bold leading-8">Peer Session Created</div>
+                <button onClick={handleClose}><Image src={closeIconBlack} className="w-5 h-5" /></button>
```

Comment:
implement the dark theme mode for the icon
refer to other components for an idea